LaTeX/TeX.sublime-syntax

```diff
@@ -111,6 +118,46 @@ contexts:
       captures:
         1: punctuation.definition.backslash.tex
 
+  registers:
+    - match: (\\)({{registers}}){{endcs}}
+      captures:
+        1: punctuation.definition.backslash.tex
+        2: storage.type.tex
+      push:
+        - meta_scope: variable.language.tex
+        - include: register-id
+
+  register-id:
```

Comment:
If `register-id` is not to be used anywhere else, you could directly push it onto stack to avoid creating anonymous contexts.

Also added suggested `meta.register.tex`. ... needs test cases to be adjusted.

```suggestion
      push: register-id

  register-id:
    - meta_scope: meta.register.tex
```